docker/ubuntu2404_gcc.docker

```diff
@@ -42,10 +42,9 @@ RUN cmake .. \
     -DBUILD_TESTING=ON \
     -DUSE_OPENMP=OFF \
     -DUSE_HDF5=ON \
-    -DBUILD_PYTHON=ON \
-    -DBUILD_PYTHON3=ON \
     -DCMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH} \
     -DDATA_DIR=/usr/local/share/casacore/data \
+    -DBUILD_FFTPACK_DEPRECATED=ON \
```

Comment:
Not sure why you'd still want to use FFTPack and then something which is deprecated. Isn't FFTW3 the best choice nowadays? But it's fine with me.

Reply:
I simply copied `ubuntu_2004` to `ubuntu_2204`.

Reply:
OK, fine. But maybe something to consider for later.